src/components/AppNavigationItem/InputConfirmCancel.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+	<div class="app-navigation-entry__inline-input-container">
```

Comment:
That seems an overly complicated class names, but it works :stuck_out_tongue:

Reply:
Should i change it ?